ClavaLaraExtraApi/src-lara/clava/ClavaMathExtra.js

```diff
@@ -1,19 +0,0 @@
-class ClavaMathExtra {
```

Comment:
Keep file

Reply:
Will be moved to a new branch and not reapplied to the main project. This is to prevent dead-code existing or being merged into staging or master.

Reply:
Resolved in 44508bf26 with the creation of the `ClavaLaraExtraAPIs` branch.